### ISSUE TITLE
slurs over line break problem

### ISSUE THREAD

**Issue by musikboden** [paulrosen/abcjs#608]
This song has slurs over the line break, and abcjs creates an extra slur with no ending, after the line break.
(tested in "editor-synth" beta 26, Chrome Version 88.0.4324.96 (Officiell version) (x86_64)

![Skärmavbild 2021-01-28 kl  00 04 12](https://user-images.githubusercontent.com/18282700/106066302-e1b9b000-60fc-11eb-88a5-932ee2d7c46e.png)

Here's the song:

X:1
T:Good news
T:Easy tunes for two
C:Arranger: Susan Schmitt
%%score { 1 | 2 } { 3 | 4 }
L:1/8
Q:1/4=140
M:3/4
I:linebreak $
K:C
V:1 treble nm="Primo"
V:2 treble 
V:3 bass nm="Secundo"
V:4 bass 
V:1
"^Allegro" (!3!ef | !>!!5!g2 !3!e2 e2) | (!>!!3!e2 !2!d2) (!2!de | !4!f2 !2!d2 d2 | !tenuto!!5!g4) (!3!ef |
!5!g2 !3!e2 e2) | (!3!e2 !2!d2) (!3!de | !5!f2 !1!B2 B2 | !2!c4) |]
V:2
 (!3!EF | !>!!1!G2 !3!E2 E2) | (!>!!3!E2 !4!D2) (!4!DE | !2!F2 !4!D2 D2 |!tenuto!!1!G4) (!3!EF |
  !1!G2 !3!E2 E2) | (!3!E2 !4!D2) (!3!DE | !1!F2 !5!B,2 B,2 | C4) |]
V:3
 z2 | z (!4!G,!2!E,G,E,G,) | z (!4!G,!3!F,G,F,G,) | z (!4!G,!3!F,G,F,G,) | z (!4!G,!2!E,G,E,G,) |
 z (!4!G,!2!E,G,E,G,) | z (!5!A,!3!F,A,F,A,) | z (!4!G,!3!F,G,F,G,) | !4!!2![G,E,]4 |]
V:4
 z2 | !1!C,2 z2 z2 | !2!B,,2 z2 z2 | !4!G,,2 z2 z2 | !1!C,2 z2 z2 | 
 !1!C,2 z2 z2 | !5!F,,2 z2 z2 | !4!G,,2 z2 z2 |!1!C,4 |]

**Comment by paulrosen** [comment]
Thanks for the report. It looks like it happens when there are two voices. Here is a simple case:
```
X:1
T:slur-over-linebreak-two-staves
%%staffwidth 300
%%stretchlast
%%score (1 2)
L:1/4
K:C
V:1
(e |
g) |]
V:2
(E |
G) |]
```
It will be fixed soon.

**Comment by musikboden** [comment]
🙏

**Comment by paulrosen** [comment]
Fixed in 6.0.0-beta.27. If there is still a problem, comment here and I'll reopen.

**Comment by musikboden** [comment]
Still same problem in beta 27, using "editor-synth", Chrome Version 88.0.4324.96 (Officiell version) (x86_64)

**Comment by musikboden** [comment]
I changed your example till a Grand Staff and the problem occur again, so it seems it has something to do with the Grand Staff?

Like this:

X:1
T:slur-over-linebreak-two-staves
%%staffwidth 300
%%stretchlast
%%score { 1 | 2 }
L:1/4
K:C
V:1
(e |
g) |]
V:2
(E |
G) |]

**Comment by paulrosen** [comment]
fixed in 6.0.0-beta.28
